src/java/us/kbase/mobu/tester/ModuleTester.java

```diff
@@ -229,14 +230,16 @@ public void logNextLine(String line, boolean isError) {
                         }
                     }).build();
             ModuleRunVersion runver = new ModuleRunVersion(
-                    new URL("https://fakefakefakefakefake.com"),
-                    new ModuleMethod("use_set_provenance.to_set_provenance_for_tests"),
-                    "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "0.0.0", "dev");
+                    new URL("https://localhost"),
+                    new ModuleMethod(moduleName + ".to_set_provenance_for_tests"),
+                    "local-docker-image", "local", "dev");
             final DockerMountPoints mounts = new DockerMountPoints(
                     Paths.get("/kb/module/work"), Paths.get("tmp"));
+            Map<String, String> localModuleToImage = new LinkedHashMap<>();
+            localModuleToImage.put(moduleName, imageName);
```

Comment:
My personal preference would be to make a small class instead of a map that takes a ModuleMethod and the String imageName, but I'll leave that up to you.